src/dataflow/src/logging/timely.rs

```diff
@@ -180,14 +186,14 @@ pub fn construct<A: Allocate>(
                                             for event in events {
```

Comment:
this is entirely a random thought but I remember having a conversation about something adjacent to stuff like this where some changes made it so that we didn't have to keep around hashtables of the stream to replay it's negations back. looked everywhere on gh and can't find it so maybe I am misremembering